Update "Get started with NVIDIA CUDA" section

crahrig opened the pull request:
Reworded this section to better link to NVIDIA's documentation on getting started.

opbld34 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/itensornative_getd3d12resource?branch=pr-en-us-51) |
[docs/native-apis.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis?branch=pr-en-us-51) |
[docs/native-apis/ILearningModelDeviceFactoryNative.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis/ILearningModelDeviceFactoryNative.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/ilearningmodeldevicefactorynative?branch=pr-en-us-51) |
[docs/native-apis/ILearningModelDeviceFactoryNative_CreateFromD3D12CommandQueue.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis/ILearningModelDeviceFactoryNative_CreateFromD3D12CommandQueue.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/ilearningmodeldevicefactorynative_createfromd3d12commandqueue?branch=pr-en-us-51) |
[docs/native-apis/ILearningModelOperatorProviderNative.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis/ILearningModelOperatorProviderNative.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/ilearningmodeloperatorprovidernative?branch=pr-en-us-51) |
[docs/native-apis/ILearningModelOperatorProviderNative_GetRegistry.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis/ILearningModelOperatorProviderNative_GetRegistry.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/ilearningmodeloperatorprovidernative_getregistry?branch=pr-en-us-51) |
[docs/native-apis/ITensorNative.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis/ITensorNative.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/itensornative?branch=pr-en-us-51) |
[docs/native-apis/ITensorNative_GetBuffer.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis/ITensorNative_GetBuffer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/itensornative_getbuffer?branch=pr-en-us-51) |
[docs/native-apis/ITensorNative_GetD3D12Resource.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis/ITensorNative_GetD3D12Resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/itensornative_getd3d12resource?branch=pr-en-us-51) |
[docs/native-apis/ITensorStaticsNative.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis/ITensorStaticsNative.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/itensorstaticsnative?branch=pr-en-us-51) |
[docs/native-apis/ITensorStaticsNative_CreateFromD3D12Resource.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/native-apis/ITensorStaticsNative_CreateFromD3D12Resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/native-apis/itensorstaticsnative_createfromd3d12resource?branch=pr-en-us-51) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C23%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810232335144293-51%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=81Eh%2BQm7Fh2I9oJ8Xjf287QYQ6uG1JIlYTA47VJHiD4%3D&st=2018-10-23T23%3A31%3A35Z&se=2018-11-23T23%3A36%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

crahrig committed:
Update "Get started with NVIDIA CUDA" section

Reworded this section to better link to NVIDIA's documentation on getting started.
